### DISCUSSION TITLE
wrtieContract Function

### DISCUSSION THREAD

**Discussion by VasilPVR** [WalletConnect/walletconnect-monorepo#3723] [Developer Support] [answered]
Dear Wallet-Connect team, we would like to ask you about one of your wagmi Actions pointed in the documentation. In Section Web3Modal - HTML/Get started, there is a function called writeContract imported from wagmi/core. Can you please send us documentation where we can see examples how that works with your SDK. Thank you in advance.

**Comment by glitch-txs** [comment, marked as answer]
This is from Wagmi
https://wagmi.sh/core/actions/writeContract
You can use any wagmi action.

**Reply by VasilPVR** [reply]
Ok, it worked. The question is that suddenly the modal is not working. It was fine and now just stopped working. I clicked on the button and the modal didn't show up. Did you make any changes past 1-2 hours or do I have to look for the problem locally?

**Reply by glitch-txs** [reply]
Try clearing localStorage, but it should work fine, otherwise please share a MRE that we can run and test. Thanks!
